crates/biome_js_analyze/tests/specs/nursery/useTrimStartEnd/valid.js

```diff
@@ -0,0 +1,26 @@
+const foo = bar.trimStart();
+const foo = bar.trimEnd();
+bar.trimStart?.()
+bar['trimStart']()
+foo.trimStart(),
+foo.trimStart?.(),
+foo.trimEnd(),
+// Not `CallExpression`
+new foo.trimLeft();,
+// Not `MemberExpression`
+trimLeft();,
+// `callee.property` is not a `Identifier`
+foo[\'trimLeft\']();',
```

Comment:
I'm afraid I don't understand this case. And the syntax doesn't seem legal to me.